<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ButtonHostServer">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>16</y>
      <width>105</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Host Server</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonConnect">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>128</y>
      <width>101</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEditHostname">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>92</y>
      <width>229</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEditPortnumber">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>92</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>1234</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>24</x>
      <y>188</y>
      <width>365</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonBeginGame">
    <property name="geometry">
     <rect>
      <x>164</x>
      <y>16</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin Game</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
